src/Backoffice/Tasks/App/Request/UpsertTaskRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Tasks\Domain\DataTransferObjects\TaskDto;
+
+class UpsertTaskRequest extends FormRequest
+{
+    public const TITLE = 'title';
+
+    public const DESCRIPTION = 'description';
+
+    public const STATUS = 'status';
+    
+    public const EMPLOYEE_ID = 'employee_id';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::TITLE => ['required'],
+            self::DESCRIPTION => ['required'],
+            self::STATUS => ['required'],
```

Comment:
For the status here we can define an enum, so we can use predefine values.

In `App/Enums`, define `TaskStatusEnum`, like this example:

```
enum PatientConsentStatus: string
{
    case Accepted = "accepted";
    case Rejected = "rejected";
    case Pending = "pending";
}
```

 and then here

```suggestion
            self::STATUS => ['required', Rule::enum(TaskStatusEnum::class)],
```